Intersect.Client/Core/Graphics.cs

```diff
@@ -190,6 +191,9 @@ public static void DrawInGame()
                 GridSwitched = false;
             }
 
+            // Get the current time so we can pass it on to every drawing plugin call.
+            var currentTime = Timing.Global.MillisecondsUTC;
```

Comment:
Oooh, now I see why you didn't call it delta. I would 100% prefer we pass in at least the delta if nothing else. I have no opposition to passing in both, but the delta is more useful if someone is adding a plugin that has any sort of movement.